components/restservice/scripts/set-manager-ssl.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+import argparse
+import subprocess
+
+
+DEFAULT_CONF_PATH = '/etc/nginx/conf.d/default.conf'
+HTTP_PATH = '/etc/nginx/conf.d/http-external-rest-server.cloudify'
+HTTPS_PATH = '/etc/nginx/conf.d/https-external-rest-server.cloudify'
```

Comment:
if you have the script on cloudify-manager-blueprints repo, then you don't have to literally write the consts here.
I think it would be better to render them while deploying the script.
Checkout the preconfigure.py of nginx.

Reply:
this is a good idea, but looks like overkill right now: if we wanted to render the script currently, we'd have to add the paths to runtime_properties, and it feels wrong to add things there to just render paths...

also, other places (like eg. nginx's default.conf, and well preconfigure.py as well) also have those paths just as strings hardcoded in them; so I think, if we want to make those paths configurable, let's make it as a separate issue, addressing all of these places (and this one as well)